tmt/base.py

```diff
@@ -1567,6 +1575,11 @@ def __init__(
             raise_on_validation_error=raise_on_validation_error,
             **kwargs)
 
+        # TODO: there is a bug in handling internal fields with `default_factory`
```

Comment:
@happz this is planned to be addressed somewhere?

Reply:
Filed and issue, referenced in https://github.com/teemtee/tmt/issues/2630